src/components/ConnectButton.tsx

```diff
@@ -68,15 +67,6 @@ export const ConnectButton = () => {
                 return <Chip onClick={openChainModal}>Wrong network</Chip>;
               }
 
-              if (!isRegistered) {
-                return (
-                  <SignupButton
-                    loading={isRegistered === undefined}
-                    onClick={onSignup}
-                  />
-                );
-              }
```

Comment:
Why did you remove it?

Reply:
because I move it to `ConnectedDetails`.
why doing this is because currently, if I connect my wallet, and I don't have attestation for my current address, I have no way to disconnect and reconnect another wallet address. 
which means, I wanna display two buttons on the navbar once the user connected their wallet -- one is the signup button, the other one is the user info button, which allow the user to disconnect their wallet.